jolly-jellyfish/src/page_maker/forms.py

```diff
@@ -0,0 +1,36 @@
+from django import forms
+from django.contrib.auth.forms import UserCreationForm
+
+from .models import Comment, Webpage, Template, User
+
+
+class CommentForm(forms.ModelForm):
+
+    class Meta:
+        model = Comment
+        fields = ['title', 'content']
+
+
+class WebpageForm(forms.ModelForm):
+
+    class Meta:
+        model = Webpage
+        fields = [
+            'name', 'template_used', 'user_title', 'user_text_1', 'user_text_2', 'user_text_3',
+            'user_image_1', 'user_image_2', 'user_image_3', 'user_image_4'
+        ]
+
+
+class TemplateForm(forms.ModelForm):
+
+    class Meta:
+        model = Template
+        fields = ['name', 'style_sheet']
```

Comment:
Would be cool if we could have a nicer widget for style_sheet, maybe with previews of the styles. Probably for another PR though.